setup.py

```diff
@@ -2,8 +2,36 @@
 
 import os
 import re
+import sys
 import codecs
-from setuptools import setup, find_packages
+import importlib
+from distutils.core import setup
```

Comment:
Yes, this is right. However, I have a difficulty with:
```
from distutils.command.clean import clean
```

It can be replaced (setuptools version 59.2.0) with:
```
from setuptools._distutils.command.clean import clean
```

But I tend to think this is not an ideal solution. What is your opinion please ?